proto/buf.yaml

```diff
@@ -0,0 +1,35 @@
+version: v1
+deps:
+  - buf.build/cosmos/cosmos-sdk
+lint:
+  use:
+    - DEFAULT
+  except:
```

Comment:
do we have a reference for excluding these?

& can we follow the sdk one? https://github.com/cosmos/cosmos-sdk/blob/master/proto/buf.yaml

we can exclude iscn if it fails, but better enforce standards to likenft

Reply:
let's exclude iscn first